apps/remix-ide-e2e/src/tests/workspace.test.ts

```diff
@@ -415,7 +415,7 @@ module.exports = {
       })
       .click('//*[@id="workspacesMenuDropdown"]/span/i')
       .waitForElementVisible('//*[@id="workspacesMenuDropdown"]/div/ul')
-      .click('//*[@id="workspacesMenuDropdown"]/div/ul/a[4]') // rename workspace_name
+      .click('//*[@id="workspacesMenuDropdown"]/div/ul/a[3]') // rename workspace_name
```

Comment:
Would be better to use data-id select instead.

Reply:
This PR is just fixing the already present e2e. Similar e2e are there in complete file

Reply:
I know. I fixed them in another file. I think we should refactor this as well, otherwise, every time we add an item, this will fail.